pipeline/Dockerfile.test

```diff
@@ -0,0 +1,20 @@
+FROM {{ base_image.image }}
+
+COPY hailjwt/setup.py /hailjwt/
+COPY hailjwt/hailjwt /hailjwt/hailjwt/
+RUN pip3 install --no-cache-dir /hailjwt && \
+  rm -rf /hailjwt
+
+COPY batch/setup.py /batch/
+COPY batch/batch/ /batch/batch/
```

Comment:
Forward slash on batch/batch/ (src) is not needed:

https://docs.docker.com/engine/reference/builder/

"""
If <src> is a directory, the entire contents of the directory are copied, including filesystem metadata.

Note: The directory itself is not copied, just its contents.
"""